build.xml

```diff
@@ -1543,9 +1543,25 @@
     </sequential>
```

Comment:
`block` package doesn't make sense -> `bitpack` or FoR encompasses the functionality in the package.